man/systemd.network.xml

```diff
@@ -1304,6 +1311,114 @@
   </refsect1>
 
   <refsect1>
+    <title>[IPv6PrefixDelegation] Section Options</title>
+    <para>The <literal>[IPv6PrefixDelegation]</literal> section contains
+    settings for sending IPv6 Router Advertisements and whether to act as
+    a router, if enabled via the <varname>IPv6PrefixDelegation=</varname>
+    option described above. IPv6 network prefixes are defined in
+    <literal>IPv6Prefix</literal> sections.</para>
```

Comment:
<varname> instead of <literal> for con file settings please, and suffix with "=", as above